src/DssSpell.t.base.sol

```diff
@@ -169,6 +169,9 @@ contract DssSpellTestBase is DSTest, DSMath {
     FlipperMomAbstract      flipMom = FlipperMomAbstract( addr.addr("FLIPPER_MOM"));
     ClipperMomAbstract      clipMom = ClipperMomAbstract( addr.addr("CLIPPER_MOM"));
     DssAutoLineAbstract    autoLine = DssAutoLineAbstract(addr.addr("MCD_IAM_AUTO_LINE"));
+    LerpFactoryAbstract lerpFactory = LerpFactoryAbstract(addr.addr("LERP_FAB"));
+
+    // Insert spell-only addresses here
```

Comment:
I think this could be removed from here.
First because if we have spell only addresses this could go to the other test fail, secondly because most of the addresses needed for test will be in the addresses files. So really no need to leave this section here.

Reply:
Done in 716865f.